#34 add equation

Wiki13002 opened the pull request:
-equation display
-enter the result of the equation

Wiki13002 committed:
#34 add equation

-equation display
-enter the result of the equation

ani119 commented:
@Wiki13002 please reslove conflicts

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+
+public class Script : MonoBehaviour
+{
+    int x;
```
We use English name of variables and functions, please look to ours code and stay in the same pattern.

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+    int y;
+    string rownanie;
+    public Text napisRownanie;
+    public InputField odpowiedz;
```
public variables should be upper than private ones

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+    }
+    public void LosujRownanie()
+    {
+        x = Random.Range(2, 11);
```
We exclude 1*2 ? And why there is (2,11)?

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+        }
+        else
+        {
+            blad.text = "spróbuj jeszcze raz";
```
Let's start it with big letter

przemytka replied:
Capital letter* :P

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+    int x;
+    int y;
+    string rownanie;
+    public Text napisRownanie;
```
We use English name of variables and functions, please look to ours code and stay in the same pattern.

ani119 commented on podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:
```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Script : MonoBehaviour
```
Please name this script with count what it do (ofc in English)

ani119 reviewed (changes requested):
As I wrote in single comments this code needs some work, pleace use pattern which all team use - english names of variables and functions (remember about names gameobjects in scene hierarchy too). Your script should have name which suggest what the script do. Some refactoring about the variable order, as I wrote a few times the public ones should be higher. More comments are written at specific lines. Please remeber to be up to date with your branch and don't have conflicts.

Wiki13002 committed:
#34 Add equation

-equation display
-enter the result of the equation
-verification of the enter equation result

Wiki13002 committed:
#34 add equations

-equation display
-enter the result of the equation
-verification of the enter equation result

Wiki13002 committed:
Merge branch 'add-base-scenes' into #34-add-equations

Wiki13002 committed:
#34 marge with add-base-scenes

ani119 reviewed (dismissed):
Good job!

AgaMazur reviewed (changes requested):
Delete unnecessary empty lines in 'multiplication' code

Wiki13002 committed:
#34 merge with add-base-scenes

Delete unnecessary empty lines
